lib/remote_files/fog_store.rb

```diff
@@ -7,6 +7,7 @@ class FogStore < AbstractStore
 
     def store!(file)
       success = directory.files.create(store_options(file))
+      # TODO: Probably get the last modified time
```

Comment:
why do we need to get the last modified time here?

Reply:
In the future it might be nice to add the last modified time to `file` so that it has the same values as what comes out of the `files` and `retrieve!` functions. Definitely not required in this PR though.